conf/views/includes/head.njk

```diff
@@ -1,3 +1,3 @@
 <!--[if lte IE 8]><link href="{{ routes.controllers.Assets.versioned("stylesheets/application-ie-8.css").url }}" rel="stylesheet" type="text/css" /><![endif]-->
 <!--[if gt IE 8]><!--><link href="{{ routes.controllers.Assets.versioned("stylesheets/application.css").url }}" media="not speech" rel="stylesheet" type="text/css" /><!--<![endif]-->
-<script type="text/javascript" src="{{ routes.controllers.Assets.versioned("javascripts/gtm.js").url }}"></script>
\ No newline at end of file
+<script src="{{ routes.controllers.Assets.versioned("javascripts/gtm.js").url }}"></script>
```

Comment:
why remove the media type?

Reply:
"For HTML 5, it is optional. If it is not specified, it defaults to text/javascript"